DataCompareCron/src/main/java/gov/cdc/datacomparecron/service/DataCompareService.java

```diff
@@ -0,0 +1,52 @@
+package gov.cdc.datacomparecron.service;
+
+import gov.cdc.datacomparecron.service.interfaces.IDataCompareService;
+import gov.cdc.datacomparecron.service.interfaces.ITokenService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class DataCompareService implements IDataCompareService {
+    private static final Logger logger = LoggerFactory.getLogger(DataCompareService.class);
+
+    @Value("${data_exchange.endpoint_compare}")
+    private String compareEndpoint;
+
+    private final RestTemplate restTemplate;
+    private final ITokenService tokenService;
+
+    public DataCompareService(RestTemplate restTemplate, ITokenService tokenService) {
+        this.restTemplate = restTemplate;
+        this.tokenService = tokenService;
+    }
+
+    @Scheduled(cron = "${scheduler.cron}")
+    public void scheduleDataCompare() {
+        logger.info("Starting scheduled data comparison");
+        compareData(false);
```

Comment:
Create an environment varaible for runNow option, this is something user can provide when execute the app